# HG changeset patch
# User "Israel Herraiz <israel.herraiz@upm.es>"
# Date 1335762301 25200
# Branch fix_warnings
# Node ID 47c26c5f082e9fc347fbd563ce2efee4be355688
# Parent  aa6e8e8b48f12b6eeaba517b8a81dbae4b637b6e
Fix warnings with XPM icons

diff -r aa6e8e8b48f1 -r 47c26c5f082e gui/src/src.pro
--- a/gui/src/src.pro	Thu Apr 26 14:22:06 2012 -0700
+++ b/gui/src/src.pro	Sun Apr 29 22:05:01 2012 -0700
@@ -71,7 +71,7 @@
 }
 
 QMAKE_LFLAGS        += $$LFLAGS $$system(mkoctfile -p RLD_FLAG)
-QMAKE_CXXFLAGS      += $$INCFLAGS
+QMAKE_CXXFLAGS      += $$INCFLAGS -Wno-write-strings -Wno-unused-variable
 
 # Files associated with the project:
 SOURCES +=\
